packages/wonder-blocks-birthday-picker/src/components/birthday-picker.js

```diff
@@ -9,6 +9,28 @@ import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
 import Color from "@khanacademy/wonder-blocks-color";
 import {SingleSelect, OptionItem} from "@khanacademy/wonder-blocks-dropdown";
 
+export type Labels = {|
+    /**
+     * Label for displaying a validation error.
+     */
+    errorMessage: string,
+
+    /**
+     * Label for the month placeholder.
+     */
+    month: string,
+
+    /**
+     * Label for the year placeholder.
+     */
+    year: string,
+
+    /**
+     * Label for the day placeholder.
+     */
+    day: string,
```

Comment:
Should we start being more prudent about read-only-ness and mark these with `+`?